src/OmniSharp.MSBuild/MSBuildProjectSystem.cs

```diff
@@ -20,6 +15,13 @@
 using OmniSharp.MSBuild.SolutionParsing;
 using OmniSharp.Options;
 using OmniSharp.Services;
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Composition;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
```

Comment:
Sort usings with `System` namespaces on top.

Reply:
I swear my option is resetting underneath me.

Reply:
bummer 😞

Reply:
Maybe you've discovered a VS bug via dogfooding?